distribute.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -euo pipefail
+IFS=$'\n\t'
+mkdir dist
+echo "Building Way Cooler..."
```

Comment:
I'm not sure I follow, what part of this workflow isn't testable? I was expecting the distribute system to be tested, even on non-release branches. All that needs to be done to test locally (or on the travis server) is to setup the environment to compile wlc statically and then you can test it works with `distribute.sh`. If you want to test installing (which would probably be better done on the travis side, to not mess with personal configuration), then you can do that with `install.sh`. 

If you want to test actually uploading these as github releases, I think you are taking the testing a step too far (we're testing our stuff works, not github's ;P). If you _want_ you can extend these scripts to update the github releases, but right now that seems out of the scopes of these two scripts (and would be bettter suited to putting in a third script)